pkg/persistence/postgrespersister.go

```diff
@@ -170,6 +170,8 @@ func (p *PostgresPersister) retrieveEventsQuery(tableName string, criteria *mode
 	}
 	if criteria.Reverse {
 		queryBuf.WriteString(" ORDER BY timestamp DESC") // nolint: gosec
+	} else {
```

Comment:
If this is by `id`, then I think the reverse flag above should also be by `id`.  Should be fine to change since there isn't any code using the reverse flag yet.

Reply:
Yeah that makes sense, will change it.